<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="TNG-FUD" failures="1" tests="2" name="Command line test" time="0.036" errors="0" timestamp="2021-08-17T15:24:33 MESZ">
  <testcase classname="com.tngtech.jgiven.tests.FailingCasesTestNgTest" name="failing_cases_do_not_lead_to_ignored_following_cases" time="0.017">
    <failure type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException
at com.tngtech.jgiven.tests.GivenTestStage.a_failed_step(GivenTestStage.java:15)
at com.tngtech.jgiven.tests.GivenTestStage$ByteBuddy$XUBaGOF7.a_failed_step$accessor$CjHxtV3M(Unknown Source)
at com.tngtech.jgiven.tests.GivenTestStage$ByteBuddy$XUBaGOF7$auxiliary$40GaAlEj.call(Unknown Source)
at com.tngtech.jgiven.impl.intercept.StepInterceptorImpl.doIntercept(StepInterceptorImpl.java:111)
at com.tngtech.jgiven.impl.intercept.StepInterceptorImpl.intercept(StepInterceptorImpl.java:82)
at com.tngtech.jgiven.impl.intercept.ByteBuddyMethodInterceptor.interceptSuper(ByteBuddyMethodInterceptor.java:37)
at com.tngtech.jgiven.tests.GivenTestStage$ByteBuddy$XUBaGOF7.a_failed_step(Unknown Source)
at com.tngtech.jgiven.tests.FailingCasesTestNgTest.failing_cases_do_not_lead_to_ignored_following_cases(FailingCasesTestNgTest.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1508)
at com.tngtech.jgiven.testng.TestNgExecutor.execute(TestNgExecutor.java:32)
at com.tngtech.jgiven.testng.TestNgExecutor.execute(TestNgExecutor.java:41)
at com.tngtech.jgiven.testframework.WhenTestFramework.the_test_class_is_executed_with(WhenTestFramework.java:38)
at com.tngtech.jgiven.testframework.WhenTestFramework$ByteBuddy$ALE5hjgE.the_test_class_is_executed_with$accessor$ZyJuZTMD(Unknown Source)
at com.tngtech.jgiven.testframework.WhenTestFramework$ByteBuddy$ALE5hjgE$auxiliary$WuNzBZeq.call(Unknown Source)
at com.tngtech.jgiven.impl.intercept.StepInterceptorImpl.doIntercept(StepInterceptorImpl.java:111)
at com.tngtech.jgiven.impl.intercept.StepInterceptorImpl.intercept(StepInterceptorImpl.java:82)
at com.tngtech.jgiven.impl.intercept.ByteBuddyMethodInterceptor.interceptSuper(ByteBuddyMethodInterceptor.java:37)
at com.tngtech.jgiven.testframework.WhenTestFramework$ByteBuddy$ALE5hjgE.the_test_class_is_executed_with(Unknown Source)
at com.tngtech.jgiven.testng.DataProviderTestNgTest.a_scenario_with_one_failing_case_still_executes_the_following_ones(DataProviderTestNgTest.java:21)
at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
at com.tngtech.jgiven.junit.JGivenMethodRule$1.evaluate(JGivenMethodRule.java:73)
at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
at org.junit.rules.RunRules.evaluate(RunRules.java:20)
at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
at com.sun.proxy.$Proxy2.processTestClass(Unknown Source)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:830)
... Removed 54 stack frames]]>
    </failure>
  </testcase> <!-- failing_cases_do_not_lead_to_ignored_following_cases -->
  <testcase classname="com.tngtech.jgiven.tests.FailingCasesTestNgTest" name="failing_cases_do_not_lead_to_ignored_following_cases" time="0.001"/>
</testsuite> <!-- Command line test -->
